modin/pandas/test/test_groupby.py

```diff
@@ -874,7 +874,9 @@ def test_simple_col_groupby():
     )
     eval_fillna(modin_groupby, pandas_groupby)
     eval_count(modin_groupby, pandas_groupby)
-    eval_size(modin_groupby, pandas_groupby)
+    if PandasCompatVersion.CURRENT != PandasCompatVersion.LATEST:
+        # due to https://github.com/pandas-dev/pandas/issues/48738, that would fail on pandas 1.5.0
+        eval_size(modin_groupby, pandas_groupby)
```

Comment:
Should we fix this as part of a separate issue?

Reply:
Yes, I was planning on making a tracker for that, but haven't made one yet. Will do.

Reply:
Please put it here once created.

Reply:
replaced pandas tracker with Modin one